src/main/scala/com/databricks/spark/avro/SchemaConverters.scala

```diff
@@ -94,6 +114,19 @@ private object SchemaConverters {
     }
   }
 
+  def dataFrameWithAliasColumn(df : DataFrame) : DataFrame = {
```

Comment:
Is this function still needed? or is this redundant with the above listing of field names ++ aliases?